frontend/src/worker/worker.ts

```diff
@@ -88,9 +88,6 @@ export const filterUserAnnouncements = (
           // sorting based on `timestamp`.
           // in Umbra.ts `fetchAllAnnouncementFromLogs` turns `timestamp` from number to string,
           // and therefore `timestamp` can be cast to number here.
-          userAnnouncements.sort(function (a, b) {
-            return parseInt(a.timestamp) - parseInt(b.timestamp);
-          });
           completion(userAnnouncements);
```

Comment:
Why was this deleted, and guessing the comment also should have been deleted?

Reply:
Cause I think there's duplicate work, we're also sorting in `AccountReceive.vue` https://github.com/ScopeLift/umbra-protocol/blob/3b2b98d895ba7ac4365f7813a8e0b59394682391/frontend/src/pages/AccountReceive.vue#L236C11-L239C16